src/main/java/io/reactivex/exceptions/OnErrorNotImplementedException.java

```diff
@@ -48,6 +48,6 @@ public OnErrorNotImplementedException(String message, @NonNull Throwable e) {
      *          the {@code Throwable} to signal; if null, a NullPointerException is constructed
      */
     public OnErrorNotImplementedException(@NonNull Throwable e) {
-        super(e != null ? e.getMessage() : null, e != null ? e : new NullPointerException());
+        this("The onError handler in the subscribe() method is not provided. Please specify it to avoid this type of exception. Further reading: https://github.com/ReactiveX/RxJava/blob/2.x/docs/Error-Handling.md | " + (e != null ? e.getMessage() : ""), e);
```

Comment:
>Please specify it to avoid this type of exception.

I'm afraid this advice is too similar to generic error handling like this

```java
try {
  method();
} catch (Throwable t) {
  t.printStackTrace();
}
```

What about something like:

>An error occurred in the flow but no error handler was provided. Further reading: https://github.com/ReactiveX/RxJava/blob/2.x/docs/Error-Handling.md

Reply:
> "no error handler was provided" 

I'm worried users will think having or adding `doOnError` will solve the error, hence the explicit mention of `subscribe()`.

Reply:
Oh, I see

Well, I'm not against mentioning `subscribe()` as long as we don't just say to "set this error handler to avoid this exception"

We can also mention `onErrorXXX`, might be a bit overwhelming tho

Reply:
How about this:

`The exception was not handled due to missing onError handler in the subscribe() method call.`

Reply:
👍